Added terraform configuration examples for GCP facilities

sourabh2311 opened the pull request.

grastogi23 commented:
@sourabh2311 thanks for updating the PR. We need to convert these to terraform 12 syntax. I have asked @Rohan-sss1 and @chaitanyaavi to help you transition this.

sourabh2311 commented:
@grastogi23 Glad to hear that our terraform provider now supports v0.12. My original configuration script for controller creation was very short due to important feature additions by terrafrom in v0.12. Now since we have updated our provider, I have updated controller creation configuration.

sourabh2311 commented:
Merged in [this](https://github.com/avinetworks/terraform-provider-avi/pull/142).
